core/controllers/dashboard.py

```diff
@@ -175,8 +176,18 @@ def _get_intro_card_color(category):
             feedback_services.get_thread_analytics_multi(
                 exploration_ids_subscribed_to))
 
+        unresolved_answers_mapping = (
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                exploration_ids_subscribed_to))
+
         for ind, exploration in enumerate(explorations_list):
             exploration.update(feedback_thread_analytics[ind].to_dict())
+            exploration.update({
+                'num_unresolved_answers': (
+                    unresolved_answers_mapping[exploration['id']]
+                    if unresolved_answers_mapping.get(exploration['id']) else 0
```

Comment:
`unresolved_answers_mapping[exploration['id']] if exploration['id'] in unresolved_answers_mapping else 0`